whereabouts.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: whereabouts
+  version: 0.7.0
+  epoch: 0
+  description: A CNI IPAM plugin that assigns IP addresses cluster-wide
+  dependencies:
+    runtime:
+      # https://github.com/k8snetworkplumbingwg/whereabouts/blob/4a6fc53afbc7c888dc803517e9eed8cb8d400214/script/install-cni.sh
+      - busybox
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/k8snetworkplumbingwg/whereabouts
+      tag: v${{package.version}}
+      expected-commit: c599b3c211fecd0c46460a10a81876ce447f6720
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.23.0
+
+  - uses: go/build
+    with:
+      modroot: .
+      ldflags: -s -w -X github.com/k8snetworkplumbingwg/whereabouts/pkg/version.Version=${{package.version}} -X github.com/k8snetworkplumbingwg/whereabouts/pkg/version.GitSHA=$(git rev-parse HEAD) -X github.com/k8snetworkplumbingwg/whereabouts/pkg/version.GitTreeState=clean -X github.com/k8snetworkplumbingwg/whereabouts/pkg/version.ReleaseStatus=released
+      packages: cmd/whereabouts.go
+      output: whereabouts
+
+  - uses: go/build
+    with:
+      modroot: .
+      ldflags: -s -w -X github.com/k8snetworkplumbingwg/whereabouts/pkg/version.Version=${{package.version}} -X github.com/k8snetworkplumbingwg/whereabouts/pkg/version.GitSHA=$(git rev-parse HEAD) -X github.com/k8snetworkplumbingwg/whereabouts/pkg/version.GitTreeState=clean -X github.com/k8snetworkplumbingwg/whereabouts/pkg/version.ReleaseStatus=released
+      packages: cmd/controlloop/controlloop.go
+      output: controlloop
+
+  - runs: |
+      mkdir -p "${{targets.contextdir}}"/
+      install -Dm755 script/install-cni.sh "${{targets.contextdir}}"/install-cni.sh
+
+  - uses: strip
```

Comment:
Handled by `go/build`

Reply:
are you sure? I don't see any strip options here 🤔 

https://github.com/chainguard-dev/melange/blob/main/pkg/build/pipelines/go/build.yaml